src/components/wizards/new-vm/NewVmWizard.js

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import { Wizard, Button } from 'patternfly-react';
+import { CreateVmWizard } from '../create-vm/CreateVmWizard';
+import { ImportVmWizard } from '../import-vm/ImportVmWizard';
+import './NewVmWizard.css';
+
+export class NewVmWizard extends React.Component {
+
+    state = {
+      createVM: false,
+      importVM: false
+    }
+  
+    openCreateVmWizard = () => this.setState({createVM: true});
+
+    openImportVmWizard = () => this.setState({importVM: true});
+  
+    render() {
+      let wizard;
+      if(this.state.createVM) {
+        wizard = <CreateVmWizard onHide={this.props.closeWizard} {...this.props}/>
+      } else if(this.state.importVM) {
+        wizard = <ImportVmWizard onHide={this.props.closeWizard} {...this.props}/>
+      }
+      return (
+        <React.Fragment>
+          <Wizard show onHide={this.props.closeWizard}>
+            <Wizard.Header title="Create Virtual Machine" onClose={this.props.closeWizard} />
+            <Wizard.Body>
+              <Wizard.Row>
+                <Wizard.Main>
+                  <Wizard.Contents stepIndex={0} activeStepIndex={0} className="wizard-content">
+                    <button className="btn btn-link wizard-button " onClick={this.openCreateVmWizard}>
+                      <div className="col-md-12">
```

Comment:
use Col component ?

Reply:
Yup, we should use patternfly-react's `Grid`, `Row` and `Col` components instead of plain HTML/CSS markup.

Also, at some point, we'll transition to patternfly-react v4 which will implement its own grid system.

Reply:
pf-react `Button`, `Col` and `Icon` used instead